microsoft/testsuites/sysbench/sysbenchperf.py

```diff
@@ -0,0 +1,406 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List
+
+from assertpy.assertpy import fail
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+)
+from lisa.messages import TestStatus, send_sub_test_result_message
+from lisa.testsuite import TestResult
+from lisa.tools import Sysbench
+from lisa.util.process import ExecutableResult
+
+
+@dataclass
+class SysbenchTestResult:
```

Comment:
It needs to either use existing perf message or add new perf messages. The general test result is not easy to analyze results.

Reply:
Could you please explain a bit here.

Reply:
This is removed with latest push

Reply:
The result is still here. If move the assertion into the tool, it doesn't need to have.

Reply:
This SysbenchTestResult, we return to testcase for exit code purpose. I have to pass the exit code to fail the parent level testcase. I can return only exit code. Will make that change